docs/src/main/paradox/user/lang/index.md

```diff
@@ -11,3 +12,28 @@ Cloudstate user functions can be implemented in any language that supports gRPC.
 * [Go](go/index.md)
 
 @@@
+
+## Unofficial client libraries
+We are aware of external libraries implementing the **client CloudState protocol** in various other languages.
```

Comment:
@sleipnir Cloudstate was recently renamed by lightbend from Cloud**S**tate to Cloud**s**tate; hence mind the s that went from uppercase to lowercase.

Reply:
Thanks Marcel

Reply:
@ChristopherAndre … ouch, I'm so sorry, I did not mention you correctly in my last comment. Thanks for the contribution.

Reply:
Fixed in the latest version!